pkg/report/linux.go

```diff
@@ -758,25 +758,34 @@ var linuxStackParams = &stackParams{
 		"dev_emerg",
 		"program_check_exception",
 		"program_check_common",
+		"kmem_cache_alloc_trace",
```

Comment:
So far we have 0 kmem/kmalloc/slab functions in this list.
Adding just 1 now flags this as 'suspicious' for me. We either should stick with the current scheme. Or maybe move all kmem/kmalloc/slab functions here. Or have some explanation as to why this single function is so special.

Reply:
OK, I'll drop this part for now. I need to look at more reports from `kmem_` functions, will send another PR at some point.